src/components/Login/index.tsx

```diff
@@ -71,51 +74,52 @@ export const FormComp = (props: StyleProps & FormikProps<FormValues>) => {
   )
 }
 
-interface StyleProps {
-  classes: {
-    grid: string
-    form: string
-  }
-}
-
-const styles = createStyles({
-  form: {
-    display: "flex",
-    minHeight: "100vh",
-  },
-  grid: {
-    display: "flex",
-    justifyContent: "center",
-    alignContent: "center",
-    color: "red",
-  },
-})
-
-export const LoginForm = withStyles(styles)(FormComp)
+//export const LoginForm = withStyles(styles)(FormComp)
 
 //TODO: Research purpose of this interface
-interface MyFormProps {
-  initlogin?: ""
+interface MyFormProps extends RouteComponentProps {
+  initPassword?: ""
+  initEmail?: ""
 }
 
-export const Login = withFormik<MyFormProps, FormValues>({
+export const LoginComponent = withFormik<MyFormProps, FormValues>({
   mapPropsToValues: props => ({
-    login: props.initlogin || "",
-    password: "",
+    email: props.initEmail || "",
+    password: props.initPassword || "",
   }),
 
   validationSchema: yup.object().shape({
-    login: yup.string().required("You must enter a login"),
-
+    email: yup
+      .string()
+      .required("You must enter an email")
+      .email(),
     password: yup.string().required("You must enter a  password"),
   }),
 
-  handleSubmit({ login, password }: FormikValues, { setSubmitting }) {
-    console.log(login)
-    alert(JSON.stringify({ login, password }, null, 2))
-    //Potential asynchronous code here
-    setSubmitting(false)
+  handleSubmit(
+    { email, password }: FormikValues,
+    { setSubmitting, props, resetForm },
+  ) {
+    axios
+      .post("http://localhost:5001/auth/login", {
+        email,
+        password,
+      })
+      .then(res => {
+        localStorage.setItem("jwtToken", res.data.token)
+        setAuthorization(res.data.token)
+        alert(JSON.stringify(res.data.message, null, 2))
```

Comment:
@Alexandria `alerts` are fine for debugging things locally when working through problems but I would make sure to convert them to `console.log` or remove them entirely before finishing the feature.